ajv/app.js

```diff
@@ -1,18 +1,17 @@
-const Ajv = require("ajv");
-const fs = require("fs");
-const glob = require("glob");
-const express = require("express");
+import Ajv2020 from "ajv/dist/2020.js";
+import fs from "fs";
+import glob from "glob";
+import express from "express";
+import Debug from "debug";
+
+const debug = Debug("ajv-schema-validator");
+
 const app = express();
-const debug = require("debug")("ajv-schema-validator");
-
-const ajv = new Ajv({
-  meta: false,
-  extendRefs: true,
-  unknownFormats: "ignore",
-  allErrors: false,
-  schemaId: "auto",
+
+const ajValidator = new Ajv2020({
+  allErrors: true,
```

Comment:
Minor, existing sets this to false. Might be because it was too verbose. Worth checking if we want to keep as it was.

Reply:
Looking at the logs while running it locally it doesn't seems to produce additional verbose. I believe the reason of this being true is to report all validation errors (rather than failing on the first errors) so that all validations are reported back.

Reply:
as discussed marked allErrors as false